main.tf

```diff
@@ -8,6 +8,10 @@ terraform {
   }
 }
 
+locals {
```

Comment:
@sgametrio does it work if `var.okta.enable` is false ? If terraform evaluate locals only when needed, then it's probably ok but otherwise we could have an error. I didn't test, just a question.
otherwise using the "try()" terraform function could resolve the issue (`okta_config=try(json...(...), "dummy"`) ?

Reply:
@rbapst-tamedia That's a very good input!!! I didn't think of it, no it won't work. So I'll revert it to what it was before so that we don't have custom code lying around